polyphemus/lib/etls/ipi/ipi_etls.rb

```diff
@@ -0,0 +1,61 @@
+require_relative '../project_watch_folders_etl'
+require_relative '../project_watch_files_etl'
+require_relative '../project_propagate_folder_updated_at_etl'
+require_relative './file_processors/ipi_rna_seq_attribute_processor'
+require_relative './file_processors/ipi_rna_seq_matrix_processor'
+require_relative './linkers/ipi_rna_seq_processed_files_linker'
+require_relative './linkers/ipi_rna_seq_raw_fastq_files_linker'
+require_relative './record_creators/ipi_rna_seq_and_plate_record_creator'
+require_relative '../linker_processor'
+
+class Polyphemus
+  module Ipi
+    class WatchFoldersConfig < Polyphemus::ProjectWatchFoldersConfig
```

Comment:
So here's a good place to start to understand this PR.  My goal was to have one file be the source for all metis oriented processing for the project.  The syntax and configuration is simple and consolidated. There are actually 3 separate etls that will run per project; one for watch folders, one for individual metis file changes, and the other propagating updated_at.

Each of these etls is idempotent and safe to reset.  In general, to reprocess linking, one only needs to reset the watch folders etl since it will process every matching folder's contents in bulk.